app/sections/image-with-text.tsx

```diff
@@ -0,0 +1,191 @@
+import type {
+  HydrogenComponentProps,
+  HydrogenComponentSchema,
+} from '@weaverse/hydrogen';
+import { forwardRef } from 'react';
+import { CSSProperties } from 'react';
+import { Image } from '@shopify/hydrogen';
+
+
+interface ImageWithTextProps extends HydrogenComponentProps {
+  imageFisrt: {
```

Comment:
Create child components instead of using `imageFirst` or `imageSecond` properties